config/secrets.yaml.template

```diff
@@ -2,20 +2,20 @@
 # Getting Started (https://github.com/Azure/aztk/blob/master/docs/00-getting-started.md)
 # NOTE - YAML requires a space after the colon. Ex: "batchaccountname: mybatchaccount"
 
-servicePrincipal:
-    tenantid: 
-    clientid: 
-    credential: 
-    batchaccountresourceid: 
-    storageaccountresourceid: 
+service_principal:
+    tenant_id:
+    client_id:
+    credential:
+    batch_account_resource_id:
+    storage_account_resource_id:
 
-sharedKey:
-    # batchaccountname: 
-    # batchaccountkey: 
-    # batchserviceurl: 
-    # storageaccountname: 
-    # storageaccountkey: 
-    # storageaccountsuffix: core.windows.net
+shared_key:
```

Comment:
I think this should be commented out too by default, otherwise the verification will fail.